tools/physiofit_manager/.shed.yml

```diff
@@ -0,0 +1,5 @@
+name: physiofit_manager
+owner: workflow4metabolomics
+description: Handling of physiofit input files
+categories: [Metabolomics]
+remote_repository_url: https://github.com/MetaboHUB-MetaToul-FluxoMet/PhysioFit_Data_Manager
```

Comment:
homepage_url is the thing you want to use ... remote_repo_url is this URL to this Galaxy tool